hydrolib/core/io/polyfile/models.py

```diff
@@ -76,6 +76,35 @@ class PolyObject(BaseModel):
     metadata: Metadata
     points: List[Point]
 
+    @property
+    def xy_coordinates(self) -> Tuple[List[float], List[float]]:
```

Comment:
Maybe xy_values/xyz_values instead of coordinates? Or is z also considered a coordinate?

Reply:
Good point I will update it to values.